core-domain/src/main/res/values/strings.xml

```diff
@@ -42,6 +42,8 @@
     <string name="sanitizer_jodel_name" translatable="false">Jodel</string>
     <string name="sanitizer_lazada" translatable="false">Lazada</string>
     <string name="sanitizer_linksynergy_name" translatable="false">LinkSynergy</string>
+    <string name="sanitizer_mydealz_paremeters_name" translatable="false">MyDealz Parameters</string>
+    <string name="sanitizer_mydealz_redirects_name" translatable="false">MyDealz Redirects</string>
```

Comment:
Since these two labels don't only contain a "product name", I guess they should be translated in the individual language `string.xml` files. Just copy the English translations for now. The German translations are:

* MyDealz Parameter
* MyDealz Weiterleitungen

Reply:
@jmbreuer Please don't forget the translations in `values-pl` and `values-vi` 😃

Reply:
... I don't see those...?

https://github.com/svenjacobs/leon/tree/main/core-domain/src/main/res

Reply:
You're right. These folders are only in `app/src/main` but those are not translations for sanitizers. Nevermind, ignore my last comment 😉